style/javascript.md

```diff
@@ -536,6 +537,404 @@ to replace all uses of `var` in existing files.
 
 
 -----------------
+### Move off Underscore
+
+Now that we're using ES6/7 many of the features of Underscore.js are now built in to the language! We should move away from using Underscore in our code towards making use of these native features.
```

Comment:
Again, the POV of this doc is telling new coders what to do, so rather than stating this as a corporate goal for our entire codebase, state it as an action item for an individual.  Something like: "Prefer using these native features to using Underscore."